packages/client/tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
```

Comment:
praise: thanks for extracting the config to the file!

question: how do you think, does it deserve its own package? It could export a function with `entry` argument, so the API would be `export default defineTsup('src/*.ts')`

Reply:
i started writing a response this morning, but then noticed tsup was creating issues with duplicating exports and bundling in consumers

today i ended up removing tsup, and using bare tsc for packaging. the packages are now structured well enough that this works.